mrp_unbuild_advanced/views/mrp_unbuild_views.xml

```diff
@@ -72,6 +50,37 @@
                     </group>
                 </group>
             </xpath>
+            <xpath expr="//button[@name='action_validate']" position="after">
+                <button
+                    name="action_back_draft"
+                    string="Back to draft"
+                    type="object"
+                    states="done"
+                    class="oe_highlight"
+                    invisible="1"
+                />
```

Comment:
This fragment of view could be simply moved (and made visible) to `mrp_unbuild_form_view_manager` view, no need to make it first invisible for user view.